src/components/core/DpEditor/DpEditor.vue

```diff
@@ -479,7 +479,12 @@ export default {
         }).length === value.length
       },
       required: false,
-      default: () => ([])
+      default: () => ([{
+        matcher: { char:'$' , allowSpaces: false, startOfLine: false },
+        suggestions: [{
+          id: 'statementPublicSubmitConfirmationTextPlaceholder',
+          name: 'Vorgangsnummer' }]
+      }])
```

Comment:
```suggestion
      default: () => ([])
```
true. this was for testing purpose. Thanks for spotting that